Sources/Commands/SwiftTestTool.swift

```diff
@@ -826,13 +932,19 @@ final class ParallelTestRunner {
             let thread = Thread {
                 // Dequeue a specifier and run it till we encounter nil.
                 while let test = self.pendingTests.dequeue() {
+#if os(macOS)
+                    let additionalArguments = ["-XCTest", test.specifier]
+#else
+                    let additionalArguments = [test.specifier]
+#endif
```

Comment:
I see this conditional assignment repeated at least twice, would it be better to generalize it as a function take the test specifier argument?

Reply:
I could. :) Do you feel strongly?

Reply:
I feel like duplicating this in multiple locations, especially with the `"-XCTest"` string constant, makes it error-prone. It can be changed/moved in one place in the future with other one easily forgotten and changes not synchronized, which may lead to subtle bugs, exacerbated by the fact that this behavior is platform-specific.

Reply:
Take another look?